engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -1342,6 +1343,19 @@ class demon_hunter_action_t : public Base
     ab::apply_affecting_aura( p->talent.havoc.any_means_necessary );
     ab::apply_affecting_aura( p->talent.havoc.dancing_with_fate );
 
+    ab::apply_affecting_aura( p->talent.vengeance.perfectly_balanced_glaive );
+    ab::apply_affecting_aura( p->talent.vengeance.meteoric_strikes );
+    ab::apply_affecting_aura( p->talent.vengeance.shear_fury );
+    ab::apply_affecting_aura( p->talent.vengeance.roaring_fire );
```

Comment:
Roaring Fire will have to be done manually since it's conditional.

Reply:
Even though it's setup as an `APPLY_AURA` on the spell data?

Reply:
#1 (id=1028507)  : Apply Aura (6) | Modify Healing% Based on Target Health% (354)
                   Base Value: 50 | Scaled Value: 50 | PvP Coefficient: 1.00000 | Misc Value: 2 | Target: Self (1)
                   Affected Spells: Fel Devastation (212106), Fel Devastation (333120)

This isn't an aura type we support manually applying. Since it would dynamically scale based on your health. Think I would implement it as an action_multiplier() override on fel_devastation_heal_t probably.

Reply:
I'll pull that out into a separate PR, good to know!